### ISSUE TITLE
[FR] Add needed dependencies to install instructions

### ISSUE THREAD

**Issue by yennor** [AppFlowy-IO/AppFlowy#1482]
### Description

I've wanted to try out AppFlowy, but first I needed to figure out what dependency it needs to start up. Just executing it, as described in the docs, definetely doesn't work ;-).
I'm using Debian Testing.
Missing libraries on my system were (could be more on other systems):
libkeybinder-3.0-0  libssl1.1

### Impact

People just wanting to try out appflowy without trying for a long time out how to get it running.

**Comment by annieappflowy** [comment]
Have you tried to download a package directly from https://github.com/AppFlowy-IO/AppFlowy/releases
I’m not sure if the Linux package works for your OS though.

**Comment by yennor** [comment]
yes I did. As mentioned, I did it exactly as descriped in the install instructions. downloading it from the link you mentioned. decompressing, executing. Doesn-t work. I had to install the above mentioned dependencies. Then it worked

**Comment by annieappflowy** [comment]
Ah, got it. Thanks so much for the clarification! We will look into it and update the documentation accordingly.

**Comment by rileyhawk1417** [comment]
![Image](https://github.com/user-attachments/assets/397ab3df-f186-449e-a2c5-03c1d74423b1)

In 0.8.1 the dependency is listed in the package information, however not entirely sure if it auto-installs the dependancy when installing  from the `.deb` package in the releases. Will need to test this.
